akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -192,6 +195,11 @@ private[remote] class SystemMessageDelivery(
             }
           }
 
+          if (incarnation != outboundContext.associationState.incarnation) {
+            log.debug("Noticed new incarnation of $remoteAddressLogParam from tryResend, clear state")
```

Comment:
remoteAddressLogParam. Needs string interpolation or a {}

Reply:
And others below

Reply:
thanks